build.py

```diff
@@ -50,6 +50,9 @@ def build_assets():
 
     print("Deleting app/static/assets")
 
+    # Copy css
+    shutil.copyfile("static/src/tasklist.css", "app/static/tasklist.css")
+
```

Comment:
I think we could safely remove this step and just commit the file directly in `app/static/styles`

Reply:
Oh wait I think this is because `app/static` is ignored by .gitignore. I still think `app/static/styles` makes more sense as a path.